lib/uuid.ex

```diff
@@ -81,17 +80,18 @@ defmodule UUID do
   ## Examples
 
   ```elixir
-  iex> UUID.binary_to_string!(<<135, 13, 248, 232, 49, 7, 68, 135, 131,
-              22, 129, 224, 137, 184, 194, 207>>)
+  iex> UUID.binary_to_string!(<<135, 13, 248, 232, 49, 7, 68, 135,
+  ...>        131, 22, 129, 224, 137, 184, 194, 207>>)
```

Comment:
These are examples of `IEx` sessions. `IEx` allows expressions on multiple lines, the extra `...>` is the symbol required to indicate that an expression can be on multiple lines. For more information please see [ExUnit.DocTest](http://elixir-lang.org/docs/master/ex_unit/ExUnit.DocTest.html)

BTW, if you run `mix test` you'll see that everything is running smoothly. So, we're talking here about valid Elixir.

Another alternative is to convert the all expression in one line, but sometimes this will imply columns with more than 80 chars.

Wdyt?